recipe/meta.yaml

```diff
@@ -1,43 +1,53 @@
-{% set version = "1.2.2" %}
+{% set version = "1.2.3" %}
 
 package:
   name: mkl_random
   version: {{ version }}
 
 source:
-  url: https://github.com/IntelPython/mkl_random/archive/v{{ version }}.post1.tar.gz
-  sha256: 7f3fd565c55e9fc83bc617764ba631f716041e9c58d6805fcf864928ff2b4b33
-  patches:
-    - patches/0001-fix-sat-5359.patch
+  url: https://github.com/IntelPython/mkl_random/archive/v{{ version }}.tar.gz
+  sha256: bb2033d82f66d9a051af821a0c1d6b5506775e181b41752b4477932e077f6b03
+  # clang does not appreciate having -std=c++11 set on the C compiler.
+  # This patch disables this. CXXFLAGS is later on set to include 
+  # -std=c++11 as a replacement.
+  patches:                              # [osx]
+    - patches/0001-cpp_flags_osx.patch  # [osx]
 
 build:
-  number: 2
+  number: 0
   skip: true  # [not x86]
+  script: 
+    - CXXFLAGS="${CXXFLAGS/-std=c++14/} -std=c++11"  # [osx]
```

Comment:
Don't you need to `export` it? Also, should we do `${CXXFLAGS/-std=c++17/}` since our default C++ version set in our compilers is 17?

Reply:
gcc toolchain has -std=c++17 set by default, but clang activation doesn't set std at all. The default is used, which is -std=c++14 for clang 14.

I added export and updated the pattern to catch more cases, in case the activation script changes later on.

Reply:
I see, I thought we enforced C++17 everywhere.